### ISSUE TITLE
Error in Google Street View while auditing, OK buttons multiplying

### ISSUE THREAD

**Issue by nchowder** [ProjectSidewalk/SidewalkWebpage#1848]
I reached a point in Google Street View where the panos went black. After a few seconds, "Error in Google Street View" popped up with one OK button. After some mouse movements, there were two more OK buttons, and then four more. 
![Screenshot_2019-08-02 Project Sidewalk](https://user-images.githubusercontent.com/17211794/62401795-9808fd00-b539-11e9-9db1-e12b61c1642e.png)
![Screenshot_2019-08-02 Project Sidewalk(1)](https://user-images.githubusercontent.com/17211794/62401797-9a6b5700-b539-11e9-95b1-1d9134d05e00.png)
I am using Firefox on Linux.

**Comment by misaugstad** [comment]
Yeah this has happened before, and I can't find the issue for it. It is a super uncommon thing that can happen with the different popus :(

**Comment by jonfroehlich** [comment]
Just happened to me:

![image](https://user-images.githubusercontent.com/1621749/63400604-4231b300-c389-11e9-8b25-0dc10cb9a069.png)

![image](https://user-images.githubusercontent.com/1621749/63400622-55dd1980-c389-11e9-885c-3d01207dd655.png)

**Comment by misaugstad** [comment]
Happened again in Chicago today. Maybe there is something that we can do to better handle this type of error? It's just really hard to work on since it does seem like it's coming from Google's end and is hard to reproduce!
![Screenshot (246)](https://user-images.githubusercontent.com/6518824/233153801-7ca7581a-3caa-4bd1-9c9f-cec9d711e20b.png)
![Screenshot (247)](https://user-images.githubusercontent.com/6518824/233153807-0e873bfa-d3ab-4632-ac17-fac3c49a167c.png)
